test/tst_plots.jl

```diff
@@ -240,6 +240,8 @@ miny = -1.2796649117521434e218
 maxy = -miny
 println(scatterplot([1],[miny],xlim=[1,1],ylim=[miny,maxy],title="Don't you crash on me!"))
 
+include("../src/interface/stemplot.jl")
```

Comment:
that is never a good idea, because we don't want a user to require having to do the same thing